composer.json

```diff
@@ -65,6 +66,17 @@
     },
     "config": {
         "sort-packages": true,
-        "optimize-autoloader": true
+        "optimize-autoloader": true,
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "cweagans/composer-patches": true
+        }
+    },
+    "extra": {
+        "patches": {
+            "keboola/csv": [
+                "patches/keboola-csv-src-csvreader-php.patch"
```

Comment:
Proč nemergneš tohle, když to je připravený: https://github.com/keboola/php-csv/pull/55?